src/targets/sourceMapOverrides.ts

```diff
@@ -44,6 +45,17 @@ export class SourceMapOverrides {
       const rightPattern = sourceMapOverrides[leftPattern];
       const entryStr = `"${leftPattern}": "${rightPattern}"`;
 
+      // Normalize patterns as URLs when we can, needed for back compat: the
+      // old adapter read the literal source URLS like `webpack:///./foo.ts`,
```

Comment:
Isn't this different from `webpack:///foo.ts` though? In the case I am vaguely remembering, this refers to a relative path in the project where the other format usually has an absolute path.

Reply:
I can't recall ever having seen webpack produce an absolute path that way and glancing through their docs I don't see an obvious way to tickle it into doing so... do you have any pointers to look into that? If not I can dive a little into their code in the morning.

Reply:
it's possible that it's like a webpack circa 2016 thing. I think it's fine to leave it alone if you think projects aren't doing that now.

Reply:
It looks like this [is configurable](https://webpack.js.org/configuration/output/#outputdevtoolmodulefilenametemplate) where people could put in a `[absolute-resource-path]`. Just searching through Github though it seems like that's generally used instead of the `webpack:///` uri, rather than in addition to. I'll probably merge this now and we can see if we get reports of any issues around it.